extensions/tokenizers/src/main/java/ai/djl/huggingface/tokenizers/jni/LibUtils.java

```diff
@@ -114,4 +134,38 @@ private static Path copyJniLibraryFromClasspath(String[] libs) {
             }
         }
     }
+
+    private static void downloadJniLib(
+            Path cacheDir,
+            Path path,
+            String djlVersion,
+            String version,
+            String classifier,
+            String flavor) {
+        String url =
+                "https://publish.djl.ai/tokenizers/"
+                        + version
+                        + "/jnilib/"
+                        + djlVersion
+                        + '/'
+                        + classifier
+                        + '/'
+                        + flavor
+                        + '/'
+                        + LIB_NAME;
+        logger.info("Downloading jni {} to cache ...", url);
```

Comment:
1. We don't have cuda for windows
2. We may only want to support cuda12

Reply:
Changed.